sdk/python/kubeflow/training/utils/utils.py

```diff
@@ -117,13 +117,40 @@ def get_script_for_python_packages(
     return script_for_python_packages
 
 
+def generate_container_spec(
```

Comment:
Let's keep this func as `get_container_spec` to be consistent with `get_pod_template_spec` WDYT @deepanker13 ?